src/Rest/RestApiBrowser.php

```diff
@@ -150,11 +161,69 @@ public function addRequestHeader($name, $value)
      */
     private function removeRequestHeader($headerName)
     {
+        $headerName = strtolower($headerName);
```

Comment:
I made header's name lower case, because I have to be sure if content-type header has 'multipart/form-data' value.
HTTP/1.1 specification says that names are case-insensitive.